### PR TITLE
 NVSHAS-9537: Change builders base image to BCI - manifest portion

### PR DESCRIPTION
**Pull request by kyledong-suse**
1. Create build_fleet_v3 and build_manager_v3 with BCI base image
2. Remove all prebuild binaries for enforcer and controller

### CODE REVIEW

File: build/amd64/Dockerfile.build_fleet_v3 [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    jemalloc-devel libboost_headers1_66_0-devel && \
+    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-13 10 && \
+    update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-13 10
+
+# Install hyperscan
+RUN zypper addrepo https://download.opensuse.org/repositories/devel:tools/15.6/devel:tools.repo && \
+    zypper --non-interactive --gpg-auto-import-keys refresh && \
+    zypper install -y ragel
+
+RUN git clone https://github.com/intel/hyperscan.git && mkdir -p hyperscan/build && \
```
Comment (holyspectral) [inline build/amd64/Dockerfile.build_fleet_v3:19]: `wget https://github.com/intel/hyperscan/archive/refs/tags/v5.4.2.tar.gz` instead?

Comment (kyledong-suse) [reply build/amd64/Dockerfile.build_fleet_v3]: Thanks for the comment. Will replace hyperscan with vectorscan, so no need to change it anymore.

**Comment by kyledong-suse** [comment]
@jayhuang-suse, I have made the replacement with vectorscan in dp. I asked QA to help to run the automation test for v3. please review again with my latest update. Thanks.

**Comment by gfsuse** [comment]
looks good to me

**Review by gfsuse — APPROVED** [review]
looks good

**Pull request merged**